Updates to use torchao's updated choose_qparams_affine and quantize/dequantize_affine

jainapurva opened the pull request:
Summary: Updates to use torchao's updated choose_qparams_affine and quantize/dequantize_affine. Remove zero_point_domain dependency

Differential Revision: D75228037

metascroy commented:
@jainapurva I think this needs to be accompanied by a torchao pin bump in OSS executorch (checkout the torchao submodule in ET and bump the commit hash)

metascroy reviewed (dismissed):
Needs a pin bump

jerryzh168 commented:
@metascroy please take a look again

jainapurva committed:
Updates to use torchao's updated choose_qparams_affine and quantize/dequantize_affine (#11070)

Summary:
Updates to use torchao's updated choose_qparams_affine and quantize/dequantize_affine. Remove zero_point_domain dependency

Pull Request resolved: https://github.com/pytorch/executorch/pull/11070

Reviewed By: mcr229

Differential Revision: D75228037

Pulled By: jerryzh168
